tensilelite/Tensile/KernelWriter.py

```diff
@@ -2622,6 +2622,14 @@ def kernelBody( self, kernel, tensorParametersA, tensorParametersB ):
         module.add(self.removeStagger(kernel, tensorParametersA))
         module.add(self.removeStagger(kernel, tensorParametersB))
 
+      supportOpt = True
```

Comment:
Maybe you can use something like "tallLoopOpt" instead of "supportOpt" to make this para clear.

Reply:
Done